src/ukify/ukify.py

```diff
@@ -356,14 +361,15 @@ def parse_input(cls, s):
             raise ValueError(f'Cannot parse section spec (extraneous parameters): {s!r}')
 
         if contents.startswith('@'):
-            contents = Path(contents[1:])
+            sec = cls.create(name, Path(contents[1:]))
+        else:
+            sec = cls.create(name, contents)
```

Comment:
Maybe collapse this to:
```python
sec = cls.create(name, Path(contents[1:]) if contents.startswith('@') else contents))
```
?

Reply:
I thought about this one as well, but I don't think it's worth it here. I think the expression is a bit too long for the ternary to be comfortable and because it hits the line length limit it would become
```python
sec = (
    cls.create(name, Path(contents[1:]) if contents.startswith('@') else contents))
)
```
without a `# fmt: skip` and that's probably too much effort for too little gain.